pom.xml

```
@@ -35,7 +35,7 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 
-		<bouncycastle.version>1.70</bouncycastle.version>
+		<bouncycastle.version>1.76</bouncycastle.version>
```

`1.77` finnes, hvis du vil være heeeeeelt bleeding edge
```suggestion
		<bouncycastle.version>1.77</bouncycastle.version>
```